### DISCUSSION TITLE
How to send Alarm when unexpected cases happened?

### DISCUSSION THREAD

**Discussion by liangliang4ward** [apache/apisix#5978] [General]
比如：某一个接口连续出现N次httpCode=5XX的情况，就产生一条告警，通知-->
或者某一个接口连续出现N次请求时间大于xxx值的情况，就产生一条告警，通知-->
类似这种规则配置有没有响应的方案可以借鉴呢？

**Comment by tzssangglass** [comment]
api-breaker

**Reply by juzhiyuan** [reply]
Hi @tzssangglass, we could use api-breaker to protect APIs from unexpected cases by breaking APIs access, and may I know how to **notify** other services when it happens?

**Reply by tzssangglass** [reply]
>  and may I know how to notify other services when it happens?

The api-breaker plugin is not supported now. However, in my experience, monitoring and alerting is generally a separate system, which is also supported by APISIX.

**Reply by liangliang4ward** [reply]
Now I use elastalert to implement this case.
log -> kafka -> es 
elastalert query es to implement alert

**Reply by tzssangglass** [reply]
> Now I use elastalert to implement this case. log -> kafka -> es elastalert query es to implement alert

It is also a way
